src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/Generated/Models/ContainerServiceOrchestratorProfile.cs

```diff
@@ -25,21 +25,31 @@ public partial class ContainerServiceOrchestratorProfile
         /// Initializes a new instance of the
         /// ContainerServiceOrchestratorProfile class.
         /// </summary>
-        /// <param name="orchestratorType">Specifies what orchestrator will be
-        /// used to manage container cluster resources. Possible values
-        /// include: 'Swarm', 'DCOS'</param>
-        public ContainerServiceOrchestratorProfile(ContainerServiceOchestratorTypes? orchestratorType = default(ContainerServiceOchestratorTypes?))
+        /// <param name="orchestratorType">The orchestrator to use to manage
+        /// container service cluster resources. Valid values are Swarm,
+        /// DCOS, and Custom. Possible values include: 'Swarm', 'DCOS',
+        /// 'Custom', 'Kubernetes'</param>
+        public ContainerServiceOrchestratorProfile(ContainerServiceOchestratorTypes orchestratorType)
```

Comment:
@brendandburns this is a breaking change - the default value for a parameter has been removed